### PR TITLE
Update publish.css

### PR DESCRIPTION
**Pull request by sigrunixia**
Update to Correct headers and P Text

Also removed !important as its not needed.

Removed some code mirrors.

### CODE REVIEW

File: publish.css [hunk 1]
```diff
@@ -1,23 +1,112 @@
...
 @import url('https://fonts.googleapis.com/css2?family=Raleway&display=swap');
 
-:root {
-  --font-monospace: 'Source Code Pro', monospace;
-
+body {
+  --font-monospace: 'Source Code Pro';
   --wws-font-modern: "Modern Antiqua";
-
   --wws-font-raleway: "Raleway";
```
Comment (sigrunixia) [inline publish.css:8]: I left this one here, but currently its not used in anything.

File: publish.css [hunk 2]
```diff
@@ -1,23 +1,112 @@
...
-}
-   
+  --wws-font-text: 'Modern Antiqua';
+  --wws-font-monospace: 'Source Code Pro', Jetbrains Mono;
+  --wws-header-font: 'Modern Antiqua';
+  --wws-header-font-caps-variant: normal;
+  --wws-font-tags: var(--wws-header-font);
+  --wws-font-callout-title: var(--wws-header-font);
+
+  --h1-font: var(--wws-header-font);
```
Comment (sigrunixia) [inline publish.css:16]: We'll use h1-h6 variables found further down in app.css to force the headers to be the font we want.

File: publish.css [hunk 3]
```diff
@@ -1,23 +1,112 @@
...
+  --font-interface-theme: var(--wws-font-text);
+  --font-mermaid: var(--wws-font-monospace);
+  }
+
 .published-container {
-     --font-text-theme: var(--wws-font-modern);
-     --page-title-variant: var(--wws-font-raleway);
-}
+  /* These are the Default Ones that Come with App.css */
+
```
Comment (sigrunixia) [inline publish.css:31]: Added the rest of these, that way you can further fine tune your text

File: publish.css [hunk 4]
```diff
@@ -85,68 +174,76 @@ Last Update:
...
-}
+  }
+
+h1,
+.markdown-rendered h1,
+.cm-line:not(.HyperMD-codeblock) .cm-header-1,
+.HyperMD-list-line .cm-header-1 {
+  font-variant-caps: var(--wws-header-font-caps-variant);
+  }
+
```
Comment (sigrunixia) [inline publish.css:185]: Added this for H1 font variant. You can add more for h2-h6. Technically, the `.cm-line:not(.HyperMD-codeblock) .cm-header-1,` can be removed.

File: publish.css [hunk 5]
```diff
@@ -160,243 +257,225 @@ Last Update:
...
-}
+  color: var(--text-a-hover);
+  text-decoration: none;
+  }
 
 mark {
-  background-color: var(--text-mark) !important;
-  color: #e5a200 !important;
-}
+  color: #e5a200;
```
Comment (sigrunixia) [inline publish.css:281]: Set to a color because text-mark is current commented out

File: publish.css [hunk 6]
```diff
@@ -160,243 +257,225 @@ Last Update:
...
-  .workspace-leaf-content,
+.workspace-split.mod-root,
+.workspace-split.mod-vertical,
+.workspace-leaf-content,
 .workspace-split.mod-vertical > .workspace-split,
 .workspace-split.mod-vertical > .workspace-leaf,
 .workspace-tabs {
-  padding-right: 0px;
-}
+  padding-right: 0;
```
Comment (sigrunixia) [inline publish.css:425]: When 0, no px is needed

File: publish.css [hunk 7]
```diff
@@ -160,243 +257,225 @@ Last Update:
...
+  height: 1px;
   background-color: var(--background-secondary-alt);
-}
+  }
 
 /* Remove vertical split padding */
-.workspace-split.mod-root
-  .workspace-split.mod-vertical
-  .workspace-leaf-content,
+.workspace-split.mod-root,
```
Comment (sigrunixia) [inline publish.css:419]: These were missing commas

File: publish.css (hunk 1 above)
Comment (Isaac-Tait) [reply publish.css:8]: Good catch, I implemented it 😄

File: publish.css (hunk 3 above)
Comment (Isaac-Tait) [reply publish.css:31]: Outstanding!

**Comment by Isaac-Tait** [comment]
@sigrunixia Thank you for your detailed pull request. It got the font working on the site now - you rock! Thank you again 💯

**Comment by Isaac-Tait** [comment]
For posterity sake https://forum.obsidian.md/t/setup-custom-font-obsidian-publish/60600/3

**Comment by sigrunixia** [comment]
> For posterity sake [forum.obsidian.md/t/setup-custom-font-obsidian-publish/60600/3](https://forum.obsidian.md/t/setup-custom-font-obsidian-publish/60600/3)

Hah, posterity. I'm using the whirlwind or the maelstrom that bugs people enough that they do whatever they can to get me off their plate (see Tabletop Games channel and https://github.com/Obsidian-TTRPG-Community). 

No clue you had an issue for so long, so glad it got resolved. :)

**Pull request merged**